#BE-3779 Docs Update - LDAP Mapping Updates (#502)
appcircleio/appcircle-docusaurus: docs/self-hosted-appcircle/configure-server/integrations-and-access/ldap-settings.md

### LDAP Synchronization

> You can use LDAP Synchronization to sync users from LDAP groups to Appcircle organizations.  The synchronization process includes adding new users and removing the unnecessary ones.

Reviewer: ```suggestion
You can synchronize users from LDAP groups to Appcircle organizations using LDAP Synchronization. This process involves adding new users and removing unnecessary ones.
```

:::info

> If the Appcircle organization is configured for synchronization, manual configurations will be overridden by the synchronization task. 

Reviewer: ```suggestion
If you configure an Appcircle organization for synchronization, the synchronization task will override any manual configurations.
```

Please note that the synchronization is one-way from LDAP to Appcircle, meaning changes made in Appcircle do not affect LDAP.

:::

:::caution

- The sync operation does not fetch all users. If a user has not logged in before, they will join the organization with the assigned roles as soon as they log in, provided LDAP Mapping is enabled.
- If a user does not exist in Appcircle (has not been imported yet), they will be ignored by the synchronization task.

> - The synchronization operation also ignores the admin user. Even if the admin user is not in the LDAP group, they can still be a member of the Appcircle organization.

Reviewer: ```suggestion
- The synchronization operation also does not affect the admin user. Even if the admin user is not in the LDAP group, they remain a member of the Appcircle organization.
```

> - Appcircle Root Organizations must at least have one owner. The synchronization operation does not remove the user if they are the last owner of the root organization.

Reviewer: ```suggestion
- Appcircle Root Organizations must have at least one owner. The synchronization operation will not remove a user if they are the last owner of the root organization.
```

> - The synchronization task needs to be run once for users who are already in Appcircle and linked to LDAP.

Reviewer: ```suggestion
- You need to run the synchronization task once for users who are already in Appcircle and linked to LDAP.
```
